config/locales/en.yml

```diff
@@ -7425,6 +7425,10 @@ en:
           attributes:
             lexicon:
               should_not_contain_commas_or_slashes: should not contain commas or slashes
+              should_match_english_translation: should match English translation, did you mean "%{suggested}"?
```

Comment:
This is a tricky concept to convey. How about "should be in English, but currently it matches the translation of "%{suggested}" into %{locale}. Use "%{suggested}" if that is what you meant and it will be translated when you view the site in that language"